### ISSUE TITLE
chaos-mesh grafana annotations are visible for all grafana datasources

### ISSUE THREAD

**Issue by csturgess93** [chaos-mesh/chaos-mesh#1482]
## Feature Request

Hi, we are using chaos mesh for some of our chaos testing and using the Grafana plugin to monitor. So far we are really enjoying it so far, it's great stuff! 

We have a slight issue that isn't a problem with Chaos Mesh, but is something we were hoping to find a workaround for.

We use a Grafana Datasource variable with our K8S cluster values inputted. This means we can alternate between each cluster on the dashboard and independently view metrics between "Cluster A" and "Cluster B" on each panel. 

The problem here is our Chaos Mesh annotation is linked to "Cluster A" namespace only and it works perfectly on Grafana when viewing "Cluster A" panels, but when I switch datasource to "Cluster B", the chaos mesh annotation visualization is still visible on all panels although not relevant as the chaos test was run on "Cluster A" and not "Cluster B". 

So in short, the chaos overlay appears on both "Cluster A" and "Cluster B" panels, which is confusing.

Do you know any work arounds/suggestions or perhaps a future feature request where we can make chaos-mesh annotations only relevant and visible for specific grafana datasources?

**Comment by Gallardot** [comment]
@yeya24  PTAL.
